src/exitstreewidget.h

```diff
@@ -54,7 +54,7 @@ class ExitsTreeWidget : public QTreeWidget
 
 public:
     Q_DISABLE_COPY(ExitsTreeWidget)
-    explicit ExitsTreeWidget(QWidget* pW);
+    explicit ExitsTreeWidget(QWidget* widget);
```

Comment:
We don't ***need*** a name here, but if we do put it lets say what it means:
```suggestion
    explicit ExitsTreeWidget(QWidget* pParent);
```